CHANGELOG.md

```diff
@@ -8,6 +8,23 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Added|Changed|Deprecated|Removed|Fixed|Security
 Nothing so far
 
+## 3.0.0 - [unreleased]
+### Added
+- Added support for php 8.
+### Removed
+- Removed everything marked deprecated in the previous major release.
+
+## 2.14.0 - 2021-03-25
+### Changed
+- Marked many functions as deprecated, i.e. `filters\match` is now available as `Filters::match`.
+  This is in preparation of the 3.0 release that will be compatible with php 8, which ships with a
+  global `match` statement of its own.
+- Marked many twig filters and functions as deprecated, i.e. `[1, 2, 3]|filter` is now available as
+  `[1, 2, 3]|it.filter` or `it([1, 2, 3]).filter`.
+  This is because twig has been adding filters that use the same names as itertools does.
+### Removed
+- Removed support for php 5, php 7.0, 7.1, 7.2, and 7.3.
```

Comment:
This comment can be moved to the 3.0.0 release.  Version 2 will remain compatible with php 5 and earlier php 7 versions